checker/tests/nullness/flow/Issue1727.java

```diff
@@ -0,0 +1,27 @@
+class B {}
+
+public class Issue1727 {
+
+    private B foo() {
+        // Default type for local variable b is @UnknownInitialization @Nullable
+        B b;
+
+        while (true) {
+            B op = getB();
+            // :: warning: (known.nonnull)
```

Comment:
Instead you could make the return type of `getB` `@Nullable B`. Otherwise the example isn't very convincing...